Makefile

```diff
@@ -184,4 +184,7 @@ ngless-${VERSION}.tar.gz: ngless
 	tar -zcvf $(distdir).tar.gz $(distdir)
 	rm -rf $(distdir)
 
-.PHONY: all build clean check tests distclean dist static fast fastcheck modules
+deploy: static
+	ngless_deploy $(shell stack path --dist-dir)/build/ngless/ngless
```

Comment:
Not intentional but also harmless. If you aren't ok with it I'll remove it from the pull request.